_sources/Unit1-Getting-Started/topic-1-6-casting.rst

```diff
@@ -100,21 +108,25 @@ For example, if you divide 5/3 using integer division, Java will truncate 1.67 t
    ~~~~
    public class NearestInt
    {
-      public static void main(String[] args)
-      {
-        double number = 5.0 / 3;
-        int nearestInt = (int)(number + 0.5);
-        System.out.println("5.0/3 = " + number);
-        System.out.println("5/3 truncated: " + (int)number );
-        System.out.println("5.0/3 rounded to nearest int: " + nearestInt);
-        double negNumber = -number;
-        int nearestNegInt = (int)(negNumber - 0.5);
-        System.out.println("-5.0/3 rounded to nearest negative int: " + nearestNegInt);
-      }
-    }
+       public static void main(String[] args)
+       {
+           double number = 5.0 / 3;
+           int nearestInt = (int) (number + 0.5);
+           System.out.println("5.0/3 = " + number);
+           System.out.println("5/3 truncated: " + (int) number);
+           System.out.println("5.0/3 rounded to nearest int: " + nearestInt);
+           double negNumber = -number;
+           int nearestNegInt = (int) (negNumber - 0.5);
+           System.out.println(
+                   "-5.0/3 rounded to nearest negative int: " + nearestNegInt);
+       }
+   }
+
     ====
```

Comment:
The } on line 123 doesn't look like it lines up with the ==== on line 125, but it might be ok.